src/main/java/academy/mate/bookstore/dto/CreateBookRequestDto.java

```diff
@@ -1,13 +1,23 @@
 package academy.mate.bookstore.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
+    @NotNull
+    @NotBlank
```

Comment:
NotNull is not required when you have NotBlank
![Screenshot 2024-01-15 210219](https://github.com/arrem4eg/Book-Store/assets/85791855/8087bbb4-0302-451e-81aa-666c92621985)